### PR TITLE
Use Trix-Editor for editing protocol.

### LINKED ISSUE #1518: Fehlende Übersetzung
screen shot 2016-01-11 at 17 34 10

### PR DESCRIPTION
**Pull request by bierik**
According to the customers the current solution to edit the protocol with markdown was not satisfactory.
We use the latets [Sablon](https://github.com/senny/sablon) version which supports HTML. As clientside editor we use [Trix](https://github.com/basecamp/trix).

Depends on https://github.com/4teamwork/plonetheme.teamraum/pull/405

bildschirmfoto 2016-01-06 um 17 25 13

Closes #1518

### CODE REVIEW

**Comment by bierik** [comment]
@deiferni updated

**Comment by bierik** [comment]
@deiferni updated

**Comment by bierik** [comment]
@deiferni rebased

File: opengever/base/upgrades/configure.zcml [hunk 1]
```diff
@@ -375,4 +375,13 @@
         profile="opengever.base:default"
         />
 
+    <!-- 4606 -> 4607 -->
+    <upgrade-step:importProfile
+        title="Resister trix javascript."
+        profile="opengever.base:default"
+        source="4606"
+        destination="4607"
```
Comment (deiferni) [inline opengever/base/upgrades/configure.zcml]: You must also increase the version opengever.base metadata.xml :bomb:!

File: opengever/base/profiles/default/jsregistry.xml [hunk 2]
```diff
@@ -44,4 +44,8 @@
                 expression="" id="++resource++opengever.base/Storage.js" inline="False"
                 insert-after="++resource++opengever.base/Controller.js" />
 
+    <javascript cacheable="True" compression="none" cookable="True" enabled="on"
+                expression="" id="++resource++opengever.base/trix.js" inline="False"
+                insert-after="++resource++opengever.meeting/sticky_heading.js" />
```
Comment (deiferni) [inline opengever/base/profiles/default/jsregistry.xml]: why is this inserted after `opengever.meeting`? you should not have a dependency on meeting stuff in `opengever.base`, you should only refer to other opengever.base stuff here, or maybe to plone/collective stuff.

File: opengever/meeting/browser/meetings/protocol.py [hunk 3]
```diff
@@ -111,6 +111,18 @@ class EditProtocol(AutoExtensibleForm, ModelEditForm):
     content_type = Meeting
     enable_unload_protection = False
 
+    agenda_item_fields = [
+        {"name": "legal_basis", "label": _("label_legal_basis", default="Legal basis"), "needs_propsal": True},
```
Comment (deiferni) [inline opengever/meeting/browser/meetings/protocol.py]: please add some newlines to improve readability :wink:

File: opengever/meeting/browser/meetings/templates/protocol.pt [hunk 4]
```diff
@@ -118,116 +61,63 @@
...
+                        <div class="button-group">
+                          <button type="button" class="button bullet" data-attribute="bullet"></button>
+                          <button type="button" class="button number" data-attribute="number"></button>
+                        </div>
+                        <div class="button-group">
+                          <button type="button" class="button increaseBlockLevel" data-action="increaseBlockLevel"></button>
+                          <button type="button" class="button decreaseBlockLevel" data-action="decreaseBlockLevel"></button>
+                        </div>
+                        <div class="button-group">
+                          <button type="button" class="button undo"data-action="undo"></button>
```
Comment (deiferni) [inline opengever/meeting/browser/meetings/templates/protocol.pt]: plz add some whitespace before the data-action attribute

**Comment by deiferni** [comment]
Die Verlinkung der Felder in der Navigation scheint bei mir nicht mehr korrekt zu Funktionieren:

screen shot 2016-01-12 at 13 25 03

**Comment by bierik** [comment]
@deiferni updated

**Comment by deiferni** [comment]
:ok_hand: :shipit:

**Pull request merged**